tests/Moq.Tests/ItIsAnyTypeFixture.cs

```diff
@@ -94,6 +94,37 @@ public void Setup_with_It_IsAnyType_default_return_value()
 			Assert.Empty((int[])result);
 		}
 
+		[Fact]
+		public void In_order_to_discover_type_arguments_Callback_accepts_a_single_IInvocation_parameter()
+		{
+			Type typeArgument = null;
+			var mock = new Mock<IZ>();
+			mock.Setup(z => z.Method<It.IsAnyType>()).Callback(new InvocationAction(invocation =>
+			{
+				typeArgument = invocation.Method.GetGenericArguments()[0];
+			}));
+
+			_ = mock.Object.Method<string>();
+
+			Assert.Equal(typeof(string), typeArgument);
+		}
+
+		[Fact]
+		public void In_order_to_discover_type_arguments_Returns_accepts_a_single_IInvocation_parameter()
```

Comment:
Should rename: `a_single_IInvocation_parameter` &rarr; `an_InvocationFunc_parameter`.